src/Constants.iOS.cs.in

```diff
@@ -147,5 +147,6 @@ namespace ObjCRuntime {
 		// iOS 14.0
 		public const string AppClipLibrary = "/System/Library/Frameworks/AppClip.framework/AppClip";
 		public const string UniformTypeIdentifiersLibrary = "/System/Library/Frameworks/UniformTypeIdentifiers.framework/UniformTypeIdentifiers";
+		public const string AccessibilityLibrary = "/System/Library/Frameworks/Accessibility.framework/Accessibility";
```

Comment:
minor: easier to check when kept in alphabetical order

Reply:
But we did not do it in src/Constants.mac.cs.in :(

Reply:
we had PR, in the past, to sort them... we might need them again (it's minor)